tap_listrak/schemas/message_sends.json

```diff
@@ -1,17 +1,61 @@
 {
-  "additionalProperties": false,
-  "properties": {
-    "MsgID": {
-      "type": [
-        "integer"
-      ]
+    "type": "object",
+    "properties": {
+        "ListID": {
+            "type": [
+                "null",
+                "integer"
+            ]
+        },
```

Comment:
Is this correct schema?
'''
<s:complexType name="WSMessageRecipient">
<s:sequence>
<s:element minOccurs="0" maxOccurs="1" name="EmailAddress" type="s:string"/>
</s:sequence>
</s:complexType>
'''